spec/interactors/api/filter_events_spec.rb

```diff
@@ -63,10 +75,14 @@
     let(:format) { :ical }
 
     it_behaves_like 'filtered by params'
-    context 'with deleted param set to true' do
-      before { params[:deleted] = true }
-      it 'always filters out deleted events' do
-        expect(events.sql).to include 'deleted'
+
+    ['true', 'all'].each do |deleted|
```

Comment:
Could you please rename the `deleted` variable to e.g. `deleted_val`? At quick look this can be confusing given the regexp below.